### DISCUSSION TITLE
[ 问题咨询 ]想咨询下手工部署的Archery服务怎样停止或重启？

### DISCUSSION THREAD

**Discussion by windyHe2020** [hhyo/Archery#1432] [Q&A]
### 问题描述
Archery目录下只有startup脚本，没有找到关闭或重启脚本。

自己尝试杀进程重启，报错：
Unlinking stale socket /usr/local/Archery/supervisor.sock

####################
（ps -ef|grep -v grep|grep manage.py|awk '{print $2}'|xargs kill -9

ps -ef|grep -v grep|grep goInception|awk '{print $2}'|xargs kill -9

ps -ef|grep supervisord|grep -v grep|awk '{print $2}'|xargs kill -9

ps -ef|grep -v grep|grep archery|awk '{print $2}'|xargs kill -9

### 版本信息
 - 应用版本/分支：Release v1.8.3  
 - 部署方式：  手工部署

**Comment by LeoQuote** [comment]
你起服务的时候是一个 supervisord , 把 supervisord 杀了, 或者用supervisord 的命令就行了, 建议自行了解下 supervisord.
